doc/design/OrionMongoSink.md

```diff
@@ -0,0 +1,110 @@
+#OrionMongoSink
```

Comment:
f you take a look on the paragraph before the section "Mapping to MongoDB data structures" you'll see a description about how HTTP notifications are converted into Flume events, which are the ones finally arriving to the sinks; such Flume events contain a destination header. I did not put all the details within this doc, but I pointed to the one giving all the details.